src/odemis/driver/xt_client.py

```diff
@@ -1010,6 +1017,21 @@ def beamlet_index_info(self):
             self.server._pyroClaimOwnership()
             return self.server.beamlet_index_info()
 
+    def scan_image(self, channel_name='electron1'):
+        """
+        Start the scan of a single image and block until the image is done scanning.
+        This function does not return the image, to get the image call get_latest_image after this function.
+
```

Comment:
Does it work also on xtlib, or only on xttoolkit? If only on xttoolkit, it'd be worth mentioning.